### PR TITLE
Need to recreate Konnected devices after update

### PR DESCRIPTION
**Pull request by haavar**

### CODE REVIEW

File: distributions/openhab/src/main/resources/bin/update.lst [hunk 1]
```diff
@@ -173,3 +173,4 @@ DELETE;$OPENHAB_USERDATA/config/org/openhab/mqttbroker.config
 
 [3.4.0]
 MOVE;$OPENHAB_USERDATA/jsondb/org.openhab.core.transform.TransformationConfiguration.json;$OPENHAB_USERDATA/jsondb/org.openhab.core.transform.Transformation.json
+ALERT;Konnected Binding: Things needs to be recreated because of the addition of Konnected Pro panel support
```
Comment (wborn) [inline distributions/openhab/src/main/resources/bin/update.lst:176]: There is a `[[MSG]]` section where it should be added instead.

**Review by haavar — COMMENTED** [review]
Moved to MSG section

Signed-off-by: Haavar Valeur <github.com@haavar.com>

File: distributions/openhab/src/main/resources/bin/update.lst [hunk 2]
```diff
@@ -172,4 +175,4 @@ DELETEDIR;$OPENHAB_USERDATA/config/org/openhab/io/metrics
 DELETE;$OPENHAB_USERDATA/config/org/openhab/mqttbroker.config
 
 [3.4.0]
-MOVE;$OPENHAB_USERDATA/jsondb/org.openhab.core.transform.TransformationConfiguration.json;$OPENHAB_USERDATA/jsondb/org.openhab.core.transform.Transformation.json
+MOVE;$OPENHAB_USERDATA/jsondb/org.openhab.core.transform.TransformationConfiguration.json;$OPENHAB_USERDATA/jsondb/org.openhab.core.transform.Transformation.json
```
Comment (wborn) [inline distributions/openhab/src/main/resources/bin/update.lst:178]: Please revert this change.

**Comment by lolodomo** [comment]
Maybe the description could be enhanced to also cover the changes done in openahb/openhab-addons#13251

**Review by haavar — COMMENTED** [review]
Added note about manual thing configuraiton

 Signed-off-by: Haavar Valeur <github.com@haavar.com>

**Review by wborn — APPROVED** [review]
Thanks!
